networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/core/Eth2OutgoingRequestHandler.java

```diff
@@ -161,7 +163,7 @@ private void cancelRequest(
       return;
     }
 
-    LOG.debug("Cancel request: {}", error.getMessage());
+    LOG.trace("Cancel request: {}", error.getMessage());
```

Comment:
This seems useful at the debug level - no?  To figure out why requests aren't processing.

Reply:
All the key cases already have a debug log before `cancelRequest` winds up getting called (and then the exception gets propagated back to the caller as well which usually winds up with more logging).